src/site-layout/SiteLayout.js

```diff
@@ -24,7 +26,7 @@ const SiteContainer = styled.div`
 
 const BodyWrapper = styled(PageWidthContainer)`
   margin: 0 auto;
-  min-height: 100vh;
+  min-height: calc(100vh - ${(props) => props.theme.footerHeight});
```

Comment:
This doesn't have any impact on your previous IE11 footer fix, right?

Reply:
I can confirm that it does not!